share/spice/tides/tides.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_tides = function(api_result){
+        if (!api_result || !api_result.tide) {
+            return Spice.failed('tides');
+        }
+
+        var now = new Date().getTime() / 1000;
+        var tides = $.grep(api_result.tide.tideSummary, function(i){
+            //filtering out events that are not high/low tides or have already happened
+            return !!i.data.height && i.date.epoch > now;
+        });
+
+        //is the tide rising or receding?
+        var rising = tides[0].data.type == "High Tide";
+
+        DDG.require('moment.js', function(){
+        var tide_display = {};
+        $.each(tides, function(i, v){
+            //time: type and height
+            var time = moment({y: v.date.year, M: v.date.mon, d: v.date.mday, h: v.date.hour, m: v.date.min}).format('ddd h:mm A');
+            tide_display[time] = v.data.type + ', ' + v.data.height;
+        })
+
+        Spice.add({
+            id: "tides",
+            name: "Tides",
+            data: {
+                title: 'Tides at ' + api_result.tide.tideInfo[0].tideSite,
+                subtitle: rising ? 'Currently rising' : 'Currently receding',
```

Comment:
I think we should flip these? The subtitle sounds like it's the more pertinent information?